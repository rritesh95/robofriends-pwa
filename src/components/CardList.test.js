import { shallow } from "enzyme";
import React from 'react';
import CardList from './CardList';

it('expect to render cardList component', () => {
    const mockRobots = [
        {
            id : 1,
            name: "John",
            email: "John@gmail.com"
        }
    ];
    expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
})