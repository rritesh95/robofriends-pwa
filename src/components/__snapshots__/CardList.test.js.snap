// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render cardList component 1`] = `
<div>
  <Card
    email="John@gmail.com"
    id={1}
    key="0"
    name="John"
  />
</div>
`;
